pkgs/tools/admin/meshcentral/default.nix

```diff
@@ -1,10 +1,10 @@
 { lib, fetchpatch, fetchzip, yarn2nix-moretea, nodejs, jq, dos2unix }:
 yarn2nix-moretea.mkYarnPackage rec {
-  version = "0.8.87";
+  version = "0.8.98";
 
   src = fetchzip {
-    url = "https://registry.npmjs.org/meshcentral/-/meshcentral-0.8.87.tgz";
-    sha256 = "1jb65pvbld83mdjdb4f4z2brqsdh3b1mvnjdhbllcsn35m705cp5";
+    url = "https://registry.npmjs.org/meshcentral/-/meshcentral-0.8.98.tgz";
```

Comment:
```suggestion
    url = "https://registry.npmjs.org/meshcentral/-/meshcentral-${version}.tgz";
```

Reply:
The update script inserts the version like this.

Reply:
Then it would be prudent to change the update script to not repeat itself unnecessarily as well.

Reply:
valid point, but out of scope for this pr. this is just a minor version bump.

if you want, you could have a look at the update-script... i think `pkgs/common-updater/scripts/update-source-version` would be a good place to start. :slightly_smiling_face:

Reply:
That is not even a custom updater which makes this change even more mundane.